src/App.js

```diff
@@ -3,13 +3,107 @@ import Form from "./Form";
 import "./App.css";
 
 class App extends React.Component {
+  constructor(){
+    super();
+    this.state = {
+      userInput: [],
+      operation: "",
+      result: "",
+    }
+  }
+
+  handleClick = (event) => {
+    const { operation, userInput } = this.state
+    event.preventDefault();
+    switch (operation) {
+      case "sum":
+        this.handleSumChange(userInput);
+        break;
+      case "average":
+        this.handleAvgChange(userInput);
+        break;
+      case "mode":
+        this.handleModeChange(userInput);
+        break;
+      default:
+        this.handleDefault();
+    }
+  }
+
+  handleUserInput = (event) => {
+    const numsInput = event.target.value.split(",").map((el) => {
+      return el;
+    })
+    this.setState({
+      userInput: numsInput,
+    });
+  }
+
+  handleOperationChange = (event) => {
+    const selectedOperator = event.target.value
+    this.setState({
+      operation: selectedOperator
+    })
+  }
+
+  handleDefault = () =>{
+    this.setState({
+      result: "Invalid input.",
+    })
+}
+
+  handleSumChange = (elArr) =>{
```

Comment:
```suggestion
 sumChange = (elArr) =>{
```
starting a method name with "handle" is usually used for **event handlers**. While you do use this sum method inside of the event handler `handleClick`, the sum method isn't itself an event handler. Event handlers are the methods that are attached _directly_ to an event. 

That being said, this kind of naming convention is subjective and up to your own personal preference, but my guess is that **_most of the time_** when other developers see a method called "handle"Something, they'll expect that it's attached _directly_ to an event, like so: `onClick={handleClick}`.